<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>simpleReview</class>
 <widget class="QMainWindow" name="simpleReview">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>619</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextBrowser" name="Txt_prob">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>731</width>
      <height>281</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="Txt_title">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>Title</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="Txt_ans">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>360</y>
      <width>731</width>
      <height>161</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="But_finish">
    <property name="geometry">
     <rect>
      <x>644</x>
      <y>540</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Next</string>
    </property>
   </widget>
   <widget class="QPushButton" name="But_show">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>540</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Show</string>
    </property>
   </widget>
   <widget class="QPushButton" name="But_extra">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>540</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Extra</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
